src/hocon_tconf.erl

```diff
@@ -1295,13 +1295,10 @@ maybe_hd(Other) -> Other.
 readable_type(T) ->
     str(hocon_schema:readable_type(T)).
 
+ensure_type_path(#{matched_type := SubType} = ErrorContext, ReadableType) ->
+    ErrorContext#{matched_type => ReadableType ++ "/" ++ SubType};
```

Comment:
I guess it's not possible for `SubType` to be `""`?

Reply:
no.